vscode/package.json

```diff
@@ -316,6 +320,11 @@
         "title": "Show circuit",
         "category": "Q#"
       },
+      {
+        "command": "qsharp-vscode.showDocumentation",
+        "title": "Show documentation",
```

Comment:
Maybe `Open API documentation` would be more informative. This could read like you're going to show documentation for the Q# language or the QDK docs.

Reply:
I don't like "open" as it suggests we open something that's already exists. Like open a file. I can thing of "Show auto-generated API documentation" or simply "Generate API documentation".

Reply:
We're not really 'generating' anything (as far as the user is concerned), just showing them docs that already exist on the code. `Show API documentation` is fine if you don't like the word `Open`.

Reply:
OK. Show API documentation it is.